### ISSUE TITLE
ClusterManager.create_clustering does not work with base ChangeHeuristic

### ISSUE THREAD

**Issue by rg687** [citp/BlockSci#258]
#### System Information (if applicable)
BlockSci version: 0.6
Using AMI: yes/no no
Total memory: 64go

Hi, 
With latest 0.6 version, it seems i cannot use heuristics as before

```python
c = blocksci.heuristics.change.optimal_change() and blocksci.heuristics.change.peeling_chain()
cm.create_clustering(cluster_data, chain, c, True)
# returns
TypeError: __call__(): incompatible function arguments. The following argument types are supported:
    1. (self: blocksci.heuristics.change.ChangeHeuristic, tx: blocksci.Tx) -> Set[blocksci.Output]

Invoked with: <blocksci.heuristics.change.ChangeHeuristic object at 0x7fa97dc04ea0>
```
and 
```python
c = blocksci.heuristics.change.none
blocksci.cluster.ClusterManager.create_clustering(cluster_data, chain, c, True)
# returns
 create_clustering(): incompatible function arguments. The following argument types are supported:
    1. (location: str, chain: blocksci.Blockchain, start: int=0, stop: int=-1, heuristic: blocksci.proxy.OptionalOutputProxy=<blocksci.proxy.OptionalOutputProxy object at 0x7fa97e67b458>, should_overwrite: bool=False, ignore_coinjoin: bool=True) -> blocksci.cluster.ClusterManager

Invoked with: '/home/ubuntu/litecoin-blocksci/clusters', <blocksci.Blockchain object at 0x7fa97de3b468>, <blocksci.heuristics.change.ChangeHeuristic object at 0x7fa71c33f1b8>, True
```

**Comment by maltemoeser** [comment]
Ah, that's broken due to an API change a while ago. For now, it will work if you use `*.unique_change` (like in [test_clustering.py#L47](https://github.com/citp/BlockSci/blob/v0.6/test/blockscipy/test_clustering.py#L47), which basically only returns a change output if it's the only one detected by the heuristic.

I'm currently in the process of rewriting the ChangeAddress API a bit, which will also fix this

**Comment by maltemoeser** [comment]
I merged #259 yesterday, which should fix this. It comes with a couple of other changes, including that the standalone heuristics now return an `OutputIterator` and we've disabled change address clustering by default (you can still specify a heuristic if you want to use it).

You'll probably still want to use `.unique_change` to make sure you only cluster the change address if you can identify a unique one though.
